assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -167,7 +169,20 @@ function PaginatedMediaGallery({
       </div>
     );
 
-  const displayLoadingPill = isMediaLoading && hasMore;
+  const [showLoadingPill, setShowLoadingPill] = useState(false);
+
+  useEffect(() => {
+    if (isMediaLoading) {
```

Comment:
Can this just be if (isMediaLoading && hasMore)? Then we can combine `displayLoadingPill` and `showLoadingPill`

Reply:
Done